runtime/doc/eval.txt

```diff
@@ -3749,6 +3750,20 @@ get({func}, {what})
 			'dict'	The dictionary
 			'args'	The list with arguments
 
+get_xdg_home([{expr}])					*get_xdg_home()*
+		Get the XDG Base directory for {expr}.
+
+		These are the directories used by Neovim loading and saving
+		config, data and cache files.
```

Comment:
How about just "These directories are the default locations for various files used by Neovim."  Then link to docs such as `vimrc` and `shada`.